src/openrct2/util/Util.h

```diff
@@ -69,4 +70,9 @@ uint8_t soft_light(uint8_t a, uint8_t b);
 
 size_t strcatftime(char* buffer, size_t bufferSize, const char* format, const struct tm* tp);
 
+template<typename E> constexpr auto to_underlying(E enumerator) noexcept
```

Comment:
We use PascalCase for functions. I also think a different name such as ```EnumValue``` might be a bit more clear.

Reply:
Yes, that makes sense. :+1: